python/tidy/HISTORY.rst

```diff
@@ -1,7 +1,16 @@
 Release History
 ---------------
 
-0.1.0 (2016-08-09)
+0.3.0 (2017-02-24)
+++++++++++++++++++
+
+- Added checking for blocked packages
+- Added checking for alphabetical order for JSON keys and ``#![feature(...)]``
+- Improved duplicated packages checking
+- Improved ordering ``use`` statements
+- And many others improvements
```

Comment:
Nit: `other`

And, you missed mine - [A lint runner for running custom lints](https://github.com/servo/servo/pull/14166) 😄